src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1996,7 +2128,16 @@ namespace Google.Example.Library.V1.Samples
                 SeriesUuid = new SeriesUuid(),
             };
             PublishSeriesResponse response = libraryServiceClient.PublishSeries(request);
-            // FIXME: inspect the results
+            // testing writing bytes field.
+            using (Stream output = File.Create($"uuid_of_series_with_book_{response.BookNames[0]}.raw"))
```

Comment:
Writing a `bytes` field can be done:
```C#
using System.IO;
using System.Linq;
...
File.WriteAllBytes($"uuid_of_series_with_book_{response.BookNames[0]}.raw", response.SeriesUuid.SeriesBytes.ToArray());
```
This will overwrite any existing file content.

Reply:
I mainly copied this from the existing sample of cloud tts: https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/4dc8fa8291dcf6fcca5d9db1788d827236ead093/texttospeech/api/TextToSpeech/TextToSpeech.cs#L165
(but I clearly missed that public method in C# should be in PascalCase, especially when generating C# code using Java -.-!)

@beccasaurus Do you have any preference on which method to use to write text/bytes to a file?

Reply:
I'd like to use `File.WriteAllBytes`. It's simplicity is wonderful for usage in code samples.

Reply:
Also +1 `using System.Linq;` for writing bytes (not needed for text) which gives us `SeriesBytes.ToArray()`

Idiomatic and clear as day ~ I <3 it